core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -3672,7 +3672,7 @@ class KafkaApis(val requestChannel: RequestChannel,
     groupCoordinator.deleteShareGroupOffsets(
       request.context,
       new DeleteShareGroupOffsetsRequestData().setGroupId(groupId).setTopics(authorizedTopics)
-    ).handle[Unit] {(responseData, exception) => {
+    ).handle[Unit] { (responseData, exception) => {
```

Comment:
Looks unrelated to this PR and the second brace looks unnecessary anyway. Maybe revert this change?

Reply:
Makes sense. I will clean it up separately in order to keep this patch clean.